components/Proposals/ProposalDetailCard.tsx

```diff
@@ -413,15 +406,10 @@ export function ProposalDetailCard({ proposalNumber }: { proposalNumber: number
           </Group>
         ) : null}
 
-        {proposal && orders ? (
-          <ProposalOrdersCard
-            markets={markets}
-            proposal={proposal}
-            orders={orders}
-            handleCrank={handleCrank}
-            isCranking={isCranking}
-          />
-        ) : null}
+        <ProposalOrdersCard
+          handleCrank={handleCrank}
```

Comment:
Shouldn't those functions and state also move to the proposal context?

Reply:
moved handleCrank to the context, not sure about handleMint, handleFinalize, handleRedeem